webviewer/src/pages/Account/components/PageWrapper/PageWrapper.styles.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+import { em } from 'styles';
+import { Typography } from 'components/_shared/Typography';
+
+export const Header = styled(Typography).attrs({
+  element: 'h2',
+  variant: 'h2'
```

Comment:
you don't need to use h$n, when you don't have  h$n-1 beforehead
h2 without h1 is not applicable

Reply:
ok, changed to h1